package.json

```diff
@@ -38,7 +39,21 @@
         "react-refresh": "0.10.0",
         "typescript": "latest",
         "@types/react": "^18.0.15",
-        "@types/react-dom": "^18.0.6",
-        "@types/styled-components": "^5.1.25"
```

Comment:
Why was this removed?

Reply:
I meant to remove only `@types/styled-components` as the package is no longer used, but in the latest commits `@types/react-dom` is re-added.

Reply:
Latest commits?  Where?  I'd remove any dependencies in a separate PR, probably wherever `@types/react-dom` is re-added.